payments/payment_manager_test.go

```diff
@@ -0,0 +1,130 @@
+package payments
```

Comment:
Could the name of this file reflect that both payment and receipt managers are being covered?

Reply:
dd4c4721